tests/unit/src/test/scala/tests/codeactions/OrganizeImportsLspSuite.scala

```diff
@@ -0,0 +1,112 @@
+package tests.codeactions
+
+import scala.meta.internal.metals.codeactions.OrganizeImports
+
+class OrganizeImportsLspSuite
+    extends BaseCodeActionLspSuite("OrganizeImports") {
+  val kind: String = OrganizeImports.kind
+  val scalacOption: List[String] = List("-Ywarn-unused-import")
+  val scalafixConf: String =
+    """|/.scalafix.conf
+       |rules = [
+       |  OrganizeImports,
+       |  ExplicitResultTypes,
+       |  RemoveUnused
+       |]
+       |
+       |ExplicitResultTypes.rewriteStructuralTypesToNamedSubclass = false
+       |
+       |RemoveUnused.imports = false
+       |
+       |OrganizeImports.groupedImports = Explode
+       |OrganizeImports.expandRelative = true
+       |OrganizeImports.removeUnused = true
+       |OrganizeImports.groups = [
+       |  "scala."
+       |  "re:javax?\\."
+       |  "*"
+       |]
+       |
+       |""".stripMargin
+
+  check(
+    "basic organize imports",
+    """
+      |package a
+      |import scala.concurrent.duration._
+      |import scala.concurrent.Future<<>>
+      |import scala.concurrent.ExecutionContext.global
+      |
+      |object A {
+      |  val d = Duration(10, MICROSECONDS)
+      |  val k = Future.successful(1)
+      |}
+      |""".stripMargin,
+    s"${OrganizeImports.title}",
+    """
+      |package a
+      |import scala.concurrent.Future
+      |import scala.concurrent.duration._
+      |
+      |object A {
+      |  val d = Duration(10, MICROSECONDS)
+      |  val k = Future.successful(1)
+      |}
+      |""".stripMargin.replace("'", "\""),
+    kind = List(kind),
+    scalacOptions = scalacOption
+  )
+
+  check(
+    "basic organize imports with existing config",
+    """
+      |package a
+      |import scala.concurrent.duration._
+      |import java.util.Optional<<>>
+      |
+      |object A {
+      |  val d = Duration(10, MICROSECONDS)
+      |  val optional = Optional.empty()
+      |}
+      |""".stripMargin,
+    s"${OrganizeImports.title}",
+    """
+      |package a
+      |import scala.concurrent.duration._
+      |
+      |import java.util.Optional
+      |
+      |object A {
+      |  val d = Duration(10, MICROSECONDS)
+      |  val optional = Optional.empty()
+      |}
+      |""".stripMargin.replace("'", "\""),
+    kind = List(kind),
+    scalafixConf = scalafixConf,
+    scalacOptions = scalacOption
+  )
+
+  check(
+    "should not work on worksheet or Ammonite files",
```

Comment:
Is this test documenting a missing feature or do we actively want to prevent this working in worksheets/ammonite files?

If that's the latter, then 👍 , otherwise I would suggest using the `.fail`.

It would look like

```scala
check("test-name".fail)
```

The main advantage of this is that we can still document the expected behavior, and explicitly say that this isn't working yet.